src/extension/analytics/send-event.js

```diff
@@ -0,0 +1,25 @@
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Send Event to Backend.
+ *
+ * @param {string} event_label
+ * @param {object} event_data
+ *
+ * @return {Promise<object>} Promise returns object
+ */
+export async function sendEvent(event_label, event_data) {
+	try {
+		return await apiFetch({
+			path: '/kb-design-library/v1/handle_event?XDEBUG_SESSION_START',
```

Comment:
Did you mean to leave this in `XDEBUG_SESSION_START`?

Reply:
Ugh, no 🤦🏼‍♂️ thanks!